src/Symfony/Component/HttpKernel/Kernel.php

```diff
@@ -595,6 +630,9 @@ protected function prepareContainer(ContainerBuilder $container)
                 $container->addObjectResource($bundle);
             }
         }
+
+        $this->build($container);
```

Comment:
I would put it after bundles, to match the common convention to register your AppBundle last, which would match making AppKernel last when calling ``build``.

Registering normal compiler passes of the project after the core normal compiler passes is more logical IMO, and would limit the need to use priority overrides.